runtime/dancebox/src/migrations.rs

```diff
@@ -563,16 +354,40 @@ where
     }
 }
 
+const IDENTITY_MIGRATION_KEY_LIMIT: u64 = u64::MAX;
+
+pub struct IdentityMigration<T>(pub PhantomData<T>);
```

Comment:
I am not sure we need this, as I think the identity pallet is first introduced in this runtime. You can verify this, and if so, you can remove this migration I think.

Also, is there any other migration we need?

Reply:
Not that I know of, other than the xcmp and dmp queues